aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -9,12 +9,14 @@ export const ThemeModeToggle = ({ active }) => {
   const label = darkMode.value ? 'Light Mode' : 'Dark Mode';
   const icon = darkMode.value ? <Sun /> : <Moon />;
   return (
-    <Button
-      active={active}
-      onClick={() => darkMode.toggle()}
-      icon={icon}
-      label={label}
-    />
+    <Tip content={label}>
+      <Button
+        id="theme-button"
```

Comment:
Can we use tip prop?

Reply:
your right good catch

Reply:
so since the label is an object I cant use the tip prop in `button` since passing children to react for as an object